libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1608,10 +1620,10 @@ void GCS_MAVLINK::send_accelcal_vehicle_position(uint32_t position)
   motors. That can be dangerous when a preflight reboot is done with
   the pilot close to the aircraft and can also damage the aircraft
```

Comment:
@peterbarker The guarded code is different I think. It will stop the IO co-processor from taking over on what is output. AFAIK, the IO processor in NavIO2 board does the IO processing, but doesn't take decisions of its own, like the one in PX4 boards - what it needs is for us to stop outputting PWM. I'm not sure why writing 0 to the RCOutput is considered a _don't update_ feature, but the workaround to use 1 (that Sub implemented) should be enough.